test/system-test/utils.py

```diff
@@ -115,9 +115,16 @@ def trust_jwt_issuer(*args):
         check=True,
     )
 
+def create_member(member_name):
+    subprocess.run(
+        ["scripts/ccf/member/create.sh", member_name],
+        cwd=REPO_ROOT,
+        check=True,
+    )
+
 def add_member(member_name):
     subprocess.run(
-        ["scripts/ccf/member/add.sh", member_name],
+        ["scripts/ccf/member/add.sh", f"{os.getenv("WORKSPACE")}/{member_name}_cert.pem"],
```

Comment:
Probably docs/Adding_members.md should be updated?

Reply:
Sure good spot

Reply:
https://github.com/microsoft/azure-privacy-sandbox-kms/pull/435/commits/139d70850c3ca620f8a428568b0a73eb314146bf